src/internal/svg/caretDownSvg.tsx

```diff
@@ -0,0 +1,19 @@
+import { fill } from '../../styles/theme';
+
+export const caretDownSvg = (
+  <svg
+    data-testid="ockTokenSelectButton_CaretDown"
```

Comment:
same here and in other places.

Reply:
okay on it

Reply:
Updating the existing data-testid fields and the aria-label fields.